lib/chain-utils/src/lib.rs

```diff
@@ -1,6 +1,6 @@
 #![feature(trait_alias)]
 
-pub mod arbitrum;
+// pub mod arbitrum;
```

Comment:
commented out for now but kept as it is currently unused but will be brought back & reused for the arbitrum voyager modules soon